src/www/ui/page/UploadVcsPage.php

```diff
@@ -135,7 +136,12 @@ protected function handleUpload(Request $request)
     $Passwd = trim($request->get('passwd'));
     $Passwd = $this->basicShEscaping($Passwd);
     if (!empty($Passwd)) {
-      $jq_args .= "--password $Passwd";
+      $jq_args .= "--password $Passwd ";
+    }
+
+    $Branch = trim(explode(' ',trim($request->get('branch')))[0]);
+    if (!empty($Branch) && !strcasecmp($VCSType,'git')) {
```

Comment:
Just a small suggestion, instead of calling `! strcasecmp()`, it will be more clear to do `strcasecmp() == 0` because on first read, I thought `$VCSType` is not `git`.

Reply:
Thanks for this comment - PR updated according to your suggestions